### PR TITLE
Implemented Cascadia support

### PR DESCRIPTION
**Pull request by aukeheerdink**
With this update [.ssl files](https://skyline.ms/wiki/home/software/BiblioSpec/page.view?name=BiblioSpec%20input%20and%20output%20file%20formats) are now also supported. There were a few more extra optional columns that I could also add, but I didn't see the relevance. But if you want me to add them let me know. The columns that are used by [cascadia](https://cascadia.readthedocs.io/en/latest/file_formats.html#output-format) are added.

### CODE REVIEW

**Review by douweschulte — CHANGES_REQUESTED** [review]
Seems good but I would argue that this is a generic file format and that should be represented in the naming and parsing (just as is done with mzTab). To properly support this as a generic format I would argue that taking up each column that is described in the format would be good. The ones with hard to parse info could just be accepted as string. The ion mobility ones can just be floating points (the units are not part of my current unit system).

**Comment by aukeheerdink** [comment]
Sure will do, what do you want me to call the generic type? ssl.rs? Or something with SpectrumSequenceList.rs, I would think ssl.rs to keep with the style of mztab?

**Comment by douweschulte** [comment]
The type can be the long version I think 'SpectsumSequenceList' butthe module name (and file name) can be ssl.rs indeed.

**Review by douweschulte — APPROVED** [review]
